docs/man_pages/bodhi.rst

```diff
@@ -179,6 +179,11 @@ The ``updates`` command allows users to interact with bodhi updates.
 
         A path to a file containing all the update details.
 
+    ``--requirements <Taskatron tasks>``
+
+        A comma or space-separated list of required Taskatron tasks that must pass for this update
+        to reach stable.
```

Comment:
@bowlofeggs All tests are targeting stable so far, but in theory we could add tests that target updates-testing in the future. That would likely require more changes, though, and is not probably anything short-term.